docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+import {
```

Comment:
You should add at least one example showing the number input with new options.  I can help with this if you like.  (Assuming you have not done this yet; I'm still reviewing of course.)

Reply:
Looks like the "Type" example seems to cover this.  Thanks!